Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_036.MySqlCredentials.ps1

```diff
@@ -4,6 +4,9 @@ Port=3306; Server=some-database-name.mysql.database.azure.com; Database=catalog_
 Server=some-database-name.mysql.database.azure.com; Database=catalog_db; Uid=username@some-database-name; Pwd=password_3; SslMode=Preferred;Port=3306; 
 Server=some-database-name.mysql.database.azure.com;Database=catalog_db; SslMode=Preferred; Pwd=password_4; Uid=username@some-database-name;
 
+# This is a malformed and invalid ADO MySQL String. UiD should be in the form of <username>@<database-name>. This should be missed.
```

Comment:
>This [](http://example.com/codeflow?start=2&length=4)

I would suggest adding in the end. Simple reason: we would not see a change in the sarif